deploy/olm-catalog/jaeger-operator/1.14.0/jaeger-operator.v1.14.0.clusterserviceversion.yaml

```diff
@@ -94,19 +94,19 @@ spec:
     * Service dependency analysis
     * Performance / latency optimization
     * OpenTracing compatible data model
-    * Multiple storage backends: Cassandra, Elasticsearch, memory.
+    * Multiple storage backends: Cassandra, Elasticsearch, Memory, Badger.
```

Comment:
My former editor used to say "Alphabetize unless there's a reason not to."  Unless these are in order of preference, we should probably update to read "Badger, Cassandra, Elasticsearch, Memory."

Reply:
Will update.